### ISSUE TITLE
truffle develop --log does not work

### ISSUE THREAD

**Issue by cds-amal** [trufflesuite/truffle#3909]
I noticed this issue referenced in a couple of closed issues and created this issue to track it.

Sorry for the inconvenience!  As a workaround you can start ganache-cli separately and use `truffle console --network development`. 

For example if truffle-config contains a network definition like:
```
module.exports = {
  networks: {
    development: {
      host: "127.0.0.1",     // Localhost (default: none)
      port: 8545,            // Standard Ethereum port (default: none)
      network_id: "*",       // Any network (default: none)
    },
  }, 
  // --- snip ---
}  
```

Start ganache in one terminal: `ganache-cli -p 8545`
Start the console in another: `truffle console --network development`

**Comment by fainashalts** [comment]
This is done merged!

**Comment by cds-amal** [comment]
This was fixed in [v5.3.2](https://github.com/trufflesuite/truffle/releases/tag/v5.3.2)
